import-export-cli/cmd/getOperationPolicies.go

```diff
@@ -0,0 +1,115 @@
+
+const getAPIPoliciesCmdLongDesc = `Display a list of API Policies in the environment specified by the flag --environment, -e`
+
+var getAPIPoliciesCmdExamples = utils.ProjectName + ` ` + GetCmdLiteral + ` ` + GetPoliciesCmdLiteral + ` ` + GetAPIPoliciesCmdLiteral + ` -e dev
+ NOTE: The flag (--environment (-e)) is mandatory`
+
+// getAPIPoliciesCmd represents the get policies api command
+var getAPIPoliciesCmd = &cobra.Command{
+	Use:     GetAPIPoliciesCmdLiteral,
+	Short:   getAPIPoliciesCmdShortDesc,
+	Long:    getAPIPoliciesCmdLongDesc,
+	Example: getAPIPoliciesCmdExamples,
+	Run: func(cmd *cobra.Command, args []string) {
+		utils.Logln(utils.LogPrefixInfo + GetAPIPoliciesCmdLiteral + " called")
+		cred, err := GetCredentials(getAPIPoliciesCmdEnvironment)
+		if err != nil {
+			utils.HandleErrorAndExit("Error getting credentials", err)
+		}
+
+		if getAllAPIPoliciesAvailable {
+			getAPIPolicyListCmdLimit = ""
+		} else {
+			if limit, err := utils.ValidateFlagWithIntegerValues(getAPIPolicyListCmdLimit); err != nil || limit == -1 {
+				if limit == -1 {
+					fmt.Println("Limit should be greater than or equal zero")
+					os.Exit(1)
+				} else if err != nil {
+					utils.HandleErrorAndExit("Error Converting Limit value", err)
+				}
+			}
+		}
+
+		executeGetAPIPoliciesCmd(cred)
+	},
+}
+
+func executeGetAPIPoliciesCmd(credential credentials.Credential) {
+	accessToken, preCommandErr := credentials.GetOAuthAccessToken(credential, getAPIPoliciesCmdEnvironment)
+
+	if preCommandErr == nil {
+
+		resp, err := impl.GetAPIPolicyListFromEnv(accessToken, getAPIPoliciesCmdEnvironment, getAPIPolicyListCmdLimit)
+		if err != nil {
+			utils.HandleErrorAndExit("Error while getting api policies", err)
+		}
+		utils.Logf(utils.LogPrefixInfo+"ResponseStatus: %v\n", resp.Status())
+
+		if resp.StatusCode() == http.StatusOK {
+			// fmt.Println("Resp: ", resp)
+			impl.PrintAPIPolicies(resp, getAPIPoliciesCmdFormat)
+		} else if resp.StatusCode() == http.StatusInternalServerError {
+			// 500 Internal Server Error
+			fmt.Println(string(resp.Body()))
+		} else {
+			// neither 200 nor 500
+			fmt.Println("Error getting API Policies:", resp.Status(), "\n", string(resp.Body()))
+		}
+	} else {
+		fmt.Println("Error getting OAuth tokens while getting API Policies:" + preCommandErr.Error())
+	}
+}
+
+func init() {
+	GetPoliciesCmd.AddCommand(getAPIPoliciesCmd)
+	getAPIPoliciesCmd.Flags().StringVarP(&getAPIPoliciesCmdEnvironment, "environment", "e",
+		"", "Environment to be searched")
+	getAPIPoliciesCmd.Flags().StringVarP(&getAPIPoliciesCmdFormat, "format", "", "", "Pretty-print API policies "+
+		"using Go Templates. Use \"{{ jsonPretty . }}\" to list all fields")
+	getAPIPoliciesCmd.Flags().StringVarP(&getAPIPolicyListCmdLimit, "limit", "l",
+		strconv.Itoa(utils.DefaultPoliciesDisplayLimit), "Maximum number of policies to return")
+	getAPIPoliciesCmd.Flags().BoolVarP(&getAllAPIPoliciesAvailable, "all", "", false, "Get all api polcies")
```

Comment:
```suggestion
	getAPIPoliciesCmd.Flags().BoolVarP(&getAllAPIPoliciesAvailable, "all", "", false, "Get all API Policies")
```
Let's start the "Policies" word with the uppercase "P"

Reply:
Check all the places

Reply:
Fixed by [02a788a](https://github.com/wso2/product-apim-tooling/pull/924/commits/02a788a1570035b84d7acfea4bc999161a1bae47)